### DISCUSSION TITLE
Export detection, id_tracking, ... to a csv or json file?

### DISCUSSION THREAD

**Discussion by abelBEDOYA** [roboflow/supervision#744] [Q&A]
Hi, 
Drawing on a video all stuff this repo does is so good. However, I don't know if it is possible to export to a .csv (or .json) detections, tracking id, positions, zone, frame, etc. Having so, one could carry out an analysis after processing a video.

Thanks.

**Comment by SkalskiP** [comment]
Hi 👋🏻 We plan to add this functionality in Q1 of 2024.

**Comment by Touville01** [comment]
Hello,

Any feedback? Plannig to try the same thing especially for the ultralytics_stream_example to detect time in zone.

**Reply by LinasKo** [reply]
Hi @Touville01 :wave: 

By now we do have a solution. Check out JSONSink and CSVSink in https://supervision.roboflow.com/0.20.0/detection/tools/save_detections/

Additionally, we're just about finished working on a cookbook with example usage. Check out the URLs in https://github.com/roboflow/supervision/pull/975

**Reply by Touville01** [reply]
Thank you
